emperor/support_files/js/controller.js

```diff
@@ -302,6 +345,61 @@ define([
 
   /**
    *
+   * Write settings file for the current controller settings
+   *
+   * The format is as follows: a javascript object with the camera position
+   * stored in the 'cameraPosition' key and the quaternion in the
+   * 'cameraQuaternion' key. Each controller in this.controllers is then saved
+   * by calling toJSON on them, and the resulting object saved under the same
+   * key as the controllers object.
+   *
+   **/
+   EmperorController.prototype.saveConfig = function() {
+    var saveinfo = {};
+    // Assuming single sceneview for now
+    sceneview = this.sceneViews[0];
+    saveinfo.cameraPosition = sceneview.camera.position;
+    saveinfo.cameraQuaternion = sceneview.camera.quaternion;
+    // Save settings for each controller in the view
+     _.each(this.controllers, function(controller, index) {
+      if (controller !== undefined) {
+        saveinfo[index] = controller.toJSON();
+      }
+    });
+
+    // Save the file
+    var blob = new Blob([JSON.stringify(saveinfo)], {type: "text/json;charset=utf-8"});
+    saveAs(blob, "emperor-settings.json");
+   };
+
+   /**
+   *
```

Comment:
Minor, but can you add one leading space before `*` to match the rest of the docstrings?